packages/storybook/stories/INTERNAL/ClosedCaptions/ClosedCaptions.stories.tsx

```diff
@@ -0,0 +1,182 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { CallComposite } from '@azure/communication-react';
+import { Stack } from '@fluentui/react';
+import { Description, Heading, Source, Subheading, Title } from '@storybook/addon-docs';
+import { Meta } from '@storybook/react/types-6-0';
+import React, { useMemo } from 'react';
+import { v1 as createGUID } from 'uuid';
+import { SingleLineBetaBanner } from '../../BetaBanners/SingleLineBetaBanner';
+import { ContosoCallContainer } from '../../CallComposite/snippets/Container.snippet';
+import { ConfigHintBanner } from '../../CallWithChatComposite/Utils';
+import {
+  COMPONENT_FOLDER_PREFIX,
+  compositeExperienceContainerStyle,
+  overviewPageImagesStackStyle
+} from '../../constants';
+import { ArgsFrom, controlsToAdd, defaultCallCompositeHiddenControls } from '../../controlsUtils';
+import { compositeLocale } from '../../localizationUtils';
+import { exampleDisableCaptions } from './ClosedCaptions';
+
+const getDocs: () => JSX.Element = () => {
+  /* eslint-disable react/no-unescaped-entities */
+  return (
+    <>
+      <Title>Closed Captions</Title>
+      <Description>
+        Azure Communication Services UI Library is adding support for Closed Captions. Closed captions is a powerful
+        tool that enables developers to enhance the accessibility of their videos. With closed captions, developers can
+        provide a textual representation of the audio content in their videos, making it easier for users who are deaf
+        or hard of hearing to follow along.
+      </Description>
+      <Description>
+        Closed Captions is enabled by default and are automatically included within the CallComposite and
+        CallWithChatComposite experiences.
+      </Description>
+
+      <Heading>ACS Based Captions</Heading>
+      <SingleLineBetaBanner topOfPage={true} />
+      <Description>
+        ACS Closed Captions are enabled by default and are automatically included within the CallComposite and
+        CallWithChatComposite experiences for calling scenarios involving ACS users only. Captions can be enabled both
+        in Mobile Web sessions and in Desktop Web sessions.
+      </Description>
+      <Description>
+        For acs captions, users can enable captions in the menu and select the spoken language for the captions.
+        Captions does not detect language automatically, so the spoken language selected needs to match the language
+        that will be used in the call. Currently, ACS captions does not support translation.
+      </Description>
+
+      <Heading>Teams Interop Closed Captions</Heading>
+      <Description>
+        Teams Interop Closed Captions is enabled by default and are automatically included within the CallComposite and
+        CallWithChatComposite experiences during a call including one or more teams users.
```

Comment:
Add somewhere this is only for Teams calls
```suggestion
        CallWithChatComposite experiences during a Teams Meeting Interop call including one or more teams users.
```